dbus-cxx-uv/CMakeLists.txt

```diff
@@ -52,7 +52,7 @@ SET(PKG_CONFIG_CFLAGS
     "-I\${includedir}"
 )
 SET(PKG_CONFIG_REQUIRES
-    "dbus-cxx-2.0, libuv"
+    "dbus-cxx-2.0, libuv-static"
```

Comment:
Long term we will want to think through how we can support both the static and dynamic complication use cases. I think this is fine for now.